tests/index/test_index_task_meilisearch.py

```diff
@@ -0,0 +1,43 @@
+# pylint: disable=invalid-name
+
+import pytest
+from tests import common
+
+def test_get_tasks_default(empty_index):
+    """Tests getting the tasks list of an empty index."""
+    task = empty_index().get_tasks()
+    assert isinstance(task, dict)
+    assert len(task) == 1
+    assert len(task['results']) == 1
+
+def test_get_tasks(empty_index, small_movies):
+    """Tests getting the tasks list of a populated index."""
+    index = empty_index()
+    response = index.add_documents(small_movies)
+    assert 'uid' in response
+    response = index.add_documents(small_movies)
+    assert 'uid' in response
+    assert 'status' in response
+    task = index.get_tasks()
+    assert len(task) == 1
+    assert len(task['results']) > 2
```

Comment:
This could to be a flaky test. With the new tasks API, previously run tasks are saved so the number of results will depend on the order of tests run.

Reply:
Thank I use your example